script-test/playercomponenttest.js

```diff
@@ -974,6 +976,54 @@ require(
           expect(mockStateUpdateCallback.calls.mostRecent().args[0].data.state).toEqual(MediaState.FATAL_ERROR);
         });
 
+        it('should publish a media state update of fatal if playback is live hls webcast with playable', function () {
+          spyOn(mockStrategy, 'getDuration').and.returnValue(100);
+          spyOn(mockStrategy, 'getCurrentTime').and.returnValue(94);
+          spyOn(mockStrategy, 'load');
+
+          setUpPlayerComponent({multiCdn: true, manifestType: 'm3u8', windowType: WindowTypes.GROWING, liveSupport: 'playable'});
+
+          mockStrategy.mockingHooks.fireErrorEvent({errorProperties: {}});
+
+          jasmine.clock().tick(5000);
+
+          expect(mockStrategy.load).toHaveBeenCalledTimes(1);
+
+          expect(mockStateUpdateCallback.calls.mostRecent().args[0].data.state).toEqual(MediaState.FATAL_ERROR);
+        });
+
+        it('should cdn failover on hls webcast with seekable', function () {
+          var secondCdn = 'b';
+          var currentTime = 10;
+          var type = 'application/dash+xml';
```

Comment:
Using DASH mime type here is confusing. I suspect you'll have to change the stub data in `setUpPlayerComponent` for this test to change it to HLS.

Reply:
Do we use `application/x-mpegURL`, or `application/vnd.apple.mpegurl`?

Reply:
`application/vnd.apple.mpegurl`